src/middlewared/middlewared/plugins/vm.py

```diff
@@ -1987,6 +2154,57 @@ def translate_device(dev):
         else:
             return False
 
+    @private
+    def get_iommu_type(self):
```

Comment:
There is no reason to keep this synchronous anymore ? I think we can just use a subprocess call for the sysctl and keep it simple ?

Reply:
Fair enough. Then I also don't need to worry about the sysctl call being used in synchronous function.

Reply:
Yup, `run` would take care of that nicely ; )

Reply:
btw just to be clear, python `sysctl` module should only be used in synchronous function and it should not be used in an `asynchronous` one as it has the potential to block the event loop